EpikCord/utils/loose.py

```diff
@@ -0,0 +1,5 @@
+def clear_none_values(d: dict):
```

Comment:
Why is this file named "loose"?

Reply:
It's going to be a file full of functions that are only a few lines and don't need a class, I don't want to make a class full of staticmethods.